types/type.go

```diff
@@ -0,0 +1,32 @@
+package types
+
+import (
+	sdk "github.com/cosmos/cosmos-sdk/types"
+)
+
+const (
+	DefaultGasLimit = uint64(210000)
+)
+
+type SendTokenRequest struct {
```

Comment:
We dont have to have such `SendTokenRequest` on day 1. Please delete them.

Reply:
removed